### PR TITLE
[DSCS-1601] Adding Pre commit hooks and total duration to works

### PR DESCRIPTION
**Pull request by danielbaker**
Adding pre-commit hooks to Amber. These are setup in Banjo and are run every time a commit is made in amber

See this banjo PR for how this can be used: https://github.com/nla/banjo/pull/1697

### CODE REVIEW

File: src/amberdb/util/DurationUtils.java [hunk 1]
```diff
@@ -37,4 +40,49 @@ public static String convertDuration(final String periodHHMMSSmm){
         }
         return newDuration;
     }
+
+    public static Float convertDurationToSeconds(String duration) {
```
Comment (danielbaker) [inline src/amberdb/util/DurationUtils.java]: Moved line-for-line from SoundFile.java

**Comment by danielbaker** [comment]
@scoen @AaronCMuller @JonathanShaw @yetti

File: src/amberdb/util/DurationUtils.java (hunk 1 above)
Comment (yetti) [inline src/amberdb/util/DurationUtils.java]: :+1:

**Comment by yetti** [comment]
Looks ok to me

**Pull request merged**